### PR TITLE
Update: Added Registration form for Django girls

### PR DESCRIPTION
**Pull request by Amanya-Brian**
The second link for "Become a coach" is copied from the Official Django girls kampala page.

### CODE REVIEW

File: app/django-girls/page.jsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default function DjangoGirls() {
                 <Link
                   className="text-white bg-pyconug-lightBlue px-12 py-4 rounded-md w-50 mt-10"
                   target="_blank"
-                  href=""
+                  href="https://forms.gle/K5nELeYzpfMR8yDf6"
```
Comment (HassanBahati) [inline app/django-girls/page.jsx:63]: @Amanya-Brian what if we had these links in the `utils/constants.js` directory and reference them by a variable name...

File: app/django-girls/page.jsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export default function DjangoGirls() {
                 <Link
                   className="text-white bg-pyconug-lightBlue px-12 py-4 rounded-md w-50 mt-10"
                   target="_blank"
-                  href=""
+                  href="https://drive.google.com/open?id=1ZxkIBG7TNS-b-9EV3S2X-x_gDWO9QuBF2QJa8ZrKzwo"
```
Comment (HassanBahati) [inline app/django-girls/page.jsx:76]: this one too

File: app/django-girls/page.jsx (hunk 1 above)
Comment (Amanya-Brian) [reply app/django-girls/page.jsx:63]: Thanks for the reminder. On it.

File: app/django-girls/page.jsx (hunk 2 above)
Comment (Amanya-Brian) [reply app/django-girls/page.jsx]: Done

**Review by HassanBahati — APPROVED** [review]
Looks good to me

**Pull request merged**